### PR TITLE
Replaces Minio refs with MinIO and links in mc docs

### PR DESCRIPTION
**Pull request by ebozduman**
Official company name `Minio` became `MinIO`. 
This change required our documentation to be modified for all references to the old name, `Minio` as `MinIO` and the links `....://xxx.minio.io` as `....://xxx.min.io` in mc documents.

## Description of the change
Here are the changes done in all `../docs/*.md` files.

Used Visual Studio Code Editor regular expressions to find and replace those patterns that match the desired text. Here are the two regular expressions used to complete the job in 2 steps:

1. Set `files to include` to `*.md`, choose `Match Case` and `Use Regular Expression` and replace the links using the following regular expressions:

| Find | Replace |
| ------ | ----------- |
| (//(slack\|docs)\\.\|//)minio\\.io | $1min.io |

The above regular expression, as can be seen, only replaced the links that start with either `slack` or `docs` or nothing before `minio.io`. That excluded `dl.minio.io` and `blog.minio.io` since their `min.io` url versions have not been setup yet.

2. 
| Find | Replace |
| ------ | ----------- |
| \bMinio\b | MinIO |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,9 +82,9 @@ mc.exe --help
 ```
 
 ## Install from Source
-Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://minio.io/downloads/#minio-client.
+Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://min.io/downloads/#minio-client.
```
Comment (eco-minio) [inline README.md]: This link 404's, `downloads` changed to `download`, we either need to change the documentation or add a redirect on the main site

File: docs/minio-admin-complete-guide.md [hunk 2]
```diff
@@ -54,17 +54,17 @@ mc.exe --help
 ```
 
 ### Install from Source
-Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://minio.io/downloads/#minio-client.
+Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://min.io/downloads/#minio-client.
```
Comment (eco-minio) [inline docs/minio-admin-complete-guide.md]: Same URL issue, change `downloads` to `download`

File: docs/minio-client-complete-guide.md [hunk 3]
```diff
@@ -77,17 +77,17 @@ mc.exe --help
 ```
 
 ### Install from Source
-Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://minio.io/downloads/#minio-client.
+Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://min.io/downloads/#minio-client.
```
Comment (eco-minio) [inline docs/minio-client-complete-guide.md]: Same URL issue as above

File: docs/zh_CN/minio-client-complete-guide.md [hunk 4]
```diff
@@ -71,17 +71,17 @@ mc.exe --help
 ```
 
 ### 通过源码安装
-通过源码安装仅适用于开发人员和高级用户。`mc update`命令不支持基于源码安装的更新通知。请从[minio-client](https://minio.io/downloads/#minio-client)下载官方版本。
+通过源码安装仅适用于开发人员和高级用户。`mc update`命令不支持基于源码安装的更新通知。请从[minio-client](https://min.io/downloads/#minio-client)下载官方版本。
```
Comment (eco-minio) [inline docs/zh_CN/minio-client-complete-guide.md]: Same URL issue

File: README_zh_CN.md [hunk 5]
```diff
@@ -76,9 +76,9 @@ mc.exe --help
 ```
 
 ## 通过源码安装
-通过源码安装仅适用于开发人员和高级用户。`mc update`命令不支持基于源码安装的更新通知。请从https://minio.io/downloads/#minio-client下载官方版本。
+通过源码安装仅适用于开发人员和高级用户。`mc update`命令不支持基于源码安装的更新通知。请从https://min.io/downloads/#minio-client下载官方版本。
```
Comment (Praveenrajmani) [inline README_zh_CN.md]: The url is unreachable. `https://min.io/download/#minio-client` is the correct one.

File: README.md (hunk 1 above)
Comment (ebozduman) [reply README.md]: Both suggestions are done.

File: docs/minio-admin-complete-guide.md (hunk 2 above)
Comment (ebozduman) [reply docs/minio-admin-complete-guide.md]: Done

File: docs/minio-client-complete-guide.md (hunk 3 above)
Comment (ebozduman) [reply docs/minio-client-complete-guide.md]: Done.

File: docs/zh_CN/minio-client-complete-guide.md (hunk 4 above)
Comment (ebozduman) [reply docs/zh_CN/minio-client-complete-guide.md]: Done.

File: README_zh_CN.md (hunk 5 above)
Comment (ebozduman) [reply README_zh_CN.md]: Done.

**Pull request merged**